pkg/crc/preflight/preflight_checks_network_linux.go

```diff
@@ -94,6 +94,13 @@ exit 0
 )
 
 var systemdResolvedPreflightChecks = [...]Check{
+	{
+		configKeySuffix:  "check-dnsmasq-networkmanager-config-file-exist",
```

Comment:
The similar keys are named `check-network-manager-config` and `check-crc-dnsmasq-file`, maybe go with `check-dnsmasq-network-manager-config`?